src/server/saved_objects/service/lib/repository.test.js

```diff
@@ -1664,12 +1696,154 @@ describe('SavedObjectsRepository', () => {
       onBeforeWrite.throws(es401);
 
       try {
-        await savedObjectsRepository.delete('type', 'id');
+        await savedObjectsRepository.delete('foo', 'id');
       } catch (error) {
         sinon.assert.calledOnce(onBeforeWrite);
         expect(error).toBe(es401);
         expect(errors.isNotAuthorizedError(error)).toBe(true);
       }
     });
   });
+
+  describe('unsupported types', () => {
```

Comment:
We're missing `create` and `update` tests.

Reply:
Wow your right, I'll add those too!